build.sbt

```diff
@@ -31,13 +34,15 @@ inThisBuild(
 )
 
 name := "jsonrpc4s"
+crossScalaVersions := List(scala212, scala213)
 releaseEarlyWith := SonatypePublisher
 publishTo := sonatypePublishToBundle.value
 libraryDependencies ++= List(
-  "io.monix" %% "monix" % "3.1.0",
-  "com.outr" %% "scribe" % "2.7.10",
-  "com.github.plokhotnyuk.jsoniter-scala" %% "jsoniter-scala-core" % "2.1.15",
-  "com.github.plokhotnyuk.jsoniter-scala" %% "jsoniter-scala-macros" % "2.1.15" % Provided,
-  "io.monix" %% "minitest" % "2.7.0" % Test,
-  "com.lihaoyi" %% "pprint" % "0.5.6" % Test
+  "io.monix" %% "monix" % "3.2.0",
+  "com.outr" %% "scribe" % "3.5.5",
+  "com.outr" %% "scribe-file" % "3.5.5" % Test,
```

Comment:
scribe split this out from the core module, so if we still want to test the file writing we need this in here.